skactiveml/pool/tests/test_typi_clust.py

```diff
@@ -0,0 +1,133 @@
+import unittest
+
+import numpy as np
+from sklearn.cluster import KMeans, DBSCAN
+from skactiveml.pool._typi_clust import TypiClust
+from skactiveml.utils import MISSING_LABEL
+from skactiveml.tests.template_query_strategy import (
+    TemplateSingleAnnotatorPoolQueryStrategy,
+)
+
+
+class TestTypiClust(
+    TemplateSingleAnnotatorPoolQueryStrategy, unittest.TestCase
+):
+    def setUp(self):
+        query_default_params_clf = {
+            "X": np.linspace(0, 1, 20).reshape(10, 2),
+            "y": np.hstack([[0, 1], np.full(8, MISSING_LABEL)]),
+        }
+        super().setUp(
+            qs_class=TypiClust,
+            init_default_params={
+                "cluster_algo_dict": {"n_init": "auto", "random_state": 0}
+            },
+            query_default_params_clf=query_default_params_clf,
```

Comment:
Please add the test also for the regression case, i.e., `query_default_params_reg=query_default_params_reg`.